sdk/servicebus/Azure.Messaging.ServiceBus/tests/Diagnostics/EventSourceTests.cs

```diff
@@ -328,7 +328,7 @@ public async Task ReceiveSingleMessageLogsEvents()
             mockTransportReceiver.Setup(
                 transportReceiver => transportReceiver.ReceiveMessagesAsync(
                     1,
-                    It.IsAny<TimeSpan?>(),
+                    It.IsAny<TimeSpan?>(), false,
```

Comment:
nit:  The formatting looks odd.   Should we consider moving `false` to it's own line for consistency?